shapely/set_operations.py

```diff
@@ -506,12 +506,10 @@ def disjoint_subset_union(a, b, **kwargs):
     :func:`union`. As such, it is recommeded to use ``disjoint_subset_union`` with
     GeometryCollections rather than individual geometries.
 
-    Requires GEOS >= 3.12.0
```

Comment:
Forgot that this gets actually added by the decorator, so removing it here to not have it duplicated.